cost/azure/schedule_instance/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.9
+
+- Changed `'.statuses[1].code/"/"|.[1]'` to `'.statuses[] | select( .code | match("PowerState")) | .code[11:50]'` so that the right item in an array is selected for status
```

Comment:
```suggestion
- Fixed bug related to Instance Status in the resulting incident data
```
Perhaps something more easily readable for changelog notes?  We can always refer to git blame / PR to understand changes in detail

Reply:
agreed